ols/constants.py

```diff
@@ -1,5 +1,12 @@
 """Constants used in business logic."""
```

Comment:
seems odd to have this at the top level as opposed to under src or utils, are you following a best practice recommendation here?

Reply:
You mean having the `ols/constants.py`? I don't think there is a best practice in terms of saying "this" should be always at "some" location.

Constants are not utils, nor specific to src (which we want to eventually want to get rid of). Constants are project-wide. Hence, they should be in the project root (`ols/`). Having multiple `constants.py` per specific directory is an option too, but I don't think it would be a better approach than a single file (it's better to have unique module names).